src/content/release/release-notes/index.md

```diff
@@ -12,9 +12,9 @@ For information about bug-fix releases,
 check out the Flutter [CHANGELOG][].
 :::
 
-* 3.38.0 _(Tentative)_
-  {% comment %} * [3.38.0 announcement][] {% endcomment %}
-  {% comment %} * [3.38.0 release notes & change log][] {% endcomment %}
+* 3.38.0
+  * [3.38.0 announcement][]
```

Comment:
```suggestion
```
Deleting. I don't want this to hold up landing the list of changes. We can add it back later...or not.